### PR TITLE
Deprecate options.mapSources and export mapSources stream

### PR DESCRIPTION
**Pull request by phated**
@nmccready How do you feel about this deprecation and recommended change?  I'm not sure why the tests are failing because they were failing when I pulled the repo down (likely a dependency change).

### CODE REVIEW

**Comment by phated** [comment]
@nmccready if you can give this just a quick sanity check (and see if you like the way I'm adding a deprecation debug message), I can get this shipped.

File: index.js [hunk 1]
```diff
@@ -2,5 +2,6 @@
 
 module.exports = {
   init: require('./src/init'),
-  write: require('./src/write')
+  write: require('./src/write'),
+  mapSources: require('@gulp-sourcemaps/map-sources')
```
Comment (nmccready) [inline index.js]: 👍

**Comment by nmccready** [comment]
Looks like travis needs to be setup. Thats it.

**Comment by nmccready** [comment]
@phated I think this line and the the next might need to be fixed to get travis working https://github.com/gulp-sourcemaps/gulp-sourcemaps/blob/master/package.json#L5 .

**Comment by phated** [comment]
I was thinking about this last night and I don't think it is the whole solution.  I'm going to dig into it a bit more.

**Comment by phated** [comment]
@nmccready the build that failed is at https://travis-ci.org/gulp-sourcemaps/gulp-sourcemaps/jobs/214467734

**Comment by phated** [comment]
@nmccready I added a test that verifies the `mapSources` stream as a suitable replacement but I'm pretty sure it needs to come before `write()` so I updated the docs.  It looks like the tests are passing now (maybe due to my `convert-source-map` changes this weekend).

**Comment by phated** [comment]
Awesome. Shipping it.

**Pull request merged**